osu.Game/Graphics/Cursor/MenuCursorContainer.cs

```diff
@@ -148,6 +168,8 @@ protected override void PopOut()
             activeCursor.ScaleTo(0.6f, 250, Easing.In);
         }
 
+        private void updateInternalVisibilityState(bool show) => fadeContainer.FadeTo(show ? 1 : 0, 120, Easing.OutQuint);
```

Comment:
Rather than applying a second different fade, let's use the same one as in `PopIn` / `PopOut` (and move the logic to a shared point of calling).

Reply:
I have played around with it, and while I've came up with a shared code that works well across different contexts, I can't say for sure if it's good or hacky:

<img width="861" alt="CleanShot 2022-10-12 at 17 47 29@2x" src="https://user-images.githubusercontent.com/22781491/195375016-200007fd-0161-44ee-bcce-d7a539a11207.png">

I've pushed it for now, but feel free to change it if you're not fond of it.

Reply:
I definitely was not fond of it. I've pushed a maybe better version, see what you think.